app/javascript/src/components/Dashboard/Notes/NotesMenu.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+
+import { Search, Plus, Settings } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+const NotesMenu = () => {
```

Comment:
Please stop with the unnecessary namespacing.

Go through https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Avoid%20Unnecessary%20Namespacing and lemme know what you have understood as the mistake over here.

Reply:
I deleted this file in my other branch and forgot to delete here while merging.

Reply:
I have renamed the function and file name but while git pull some merge conflict resulted in 2 files and I forgot to delete it.